source/extensions/filters/network/dubbo_proxy/utility.h

```diff
@@ -0,0 +1,17 @@
+#include <string>
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace DubboProxy {
+
+class Utility {
+public:
+  static bool isContainWildcard(const std::string& input);
```

Comment:
I don't see these two used, are they?

Reply:
Sorry，the utility class is useless(because the method name is changed to StringMatcher), so I delete it.